web/src/lib/user.ts

```diff
@@ -6,84 +6,84 @@ const USER_KEY = 'userdata';
 interface UserAccents {
   [key: string]: string;
```

Comment:
unrelated: hmmm... should we remove this additional key type declaration? we generally wouldn't want additional properties to be defined on these objects, right?

Reply:
since we import these interfaces in User and Profile components, and then dynamically access their various strings, we need to tell the TS compiler the type to expect when making this access. 

this was done to when we disallowed implicit anys in 087c10fc89d178708f91248d20da872942dd526c